### PR TITLE
Use BaguetteBox in base theme

### PR DESCRIPTION
**Pull request by ralsina**
This adds a nice, lightweight lightbox to the base theme.

Very conservative branch: no changes in bootstrap3, no captions, just the very barebones support.

### CODE REVIEW

File: nikola/data/themes/base/templates/base.tmpl [hunk 1]
```diff
@@ -20,7 +20,13 @@ ${template_hooks['extra_head']()}
          ${footer.html_footer()}
     </div>
     ${base.late_load_js()}
+    % if use_cdn:
+        <script src="//cdnjs.cloudflare.com/ajax/libs/baguettebox.js/1.8.2/baguetteBox.min.js"></script>
```
Comment (Kwpolska) [inline nikola/data/themes/base/templates/base.tmpl]: https://

**Comment by Kwpolska** [comment]
Are captions supported if we wanted them?

File: nikola/data/themes/base/templates/base.tmpl (hunk 1 above)
Comment (ralsina) [inline nikola/data/themes/base/templates/base.tmpl]: This keeps it protocol-relative, is https better? It used to give a mixed-content warning :-P

Comment (Kwpolska) [inline nikola/data/themes/base/templates/base.tmpl]: protocol-relative stopped being cool [a while back](https://www.paulirish.com/2010/the-protocol-relative-url/). Using httpS never gave mixed-content warnings, only http did.

**Comment by ralsina** [comment]
@Kwpolska yes, they are

File: nikola/data/themes/base/templates/base.tmpl (hunk 1 above)
Comment (ralsina) [inline nikola/data/themes/base/templates/base.tmpl]: Ok, https it is

**Comment by ralsina** [comment]
I have NO IDEA why this is failing.

**Comment by Kwpolska** [comment]
(PS. CSSLint is pretty crazy, so I nuked it.)

**Pull request merged**